creator-node/tsconfig.json

```diff
@@ -0,0 +1,32 @@
+{
+  "compilerOptions": {
+    "baseUrl": "./src",
+    "target": "es5",
```

Comment:
any reason we transpile down to ES5? If we run on machines supporting node 14 (my understanding) we [could do `es2020`](https://stackoverflow.com/questions/61305578/what-typescript-configuration-produces-output-closest-to-node-js-14-capabilities) as a target.

Reply:
i'll try es2020